### DISCUSSION TITLE
Log Contacts to All Clients, Not Just the Aggregation Server

### DISCUSSION THREAD

**Discussion by kybrjo** [mbridak/FieldDayLogger#31] [Ideas]
Is it possible to have logged contacts from one client be logged on all other clients, too?  That would let each client 'know' about all contacts and warn of duplicates from other clients.

Maybe this functionality exists but I couldn't figure out how to make it work.

**Comment by mbridak** [comment]
I'd have to noodle that a bit. 
I think I'd rather ping the aggregation server with a packet asking if it's a dupe. 

Mike K6GTE

**Reply by kybrjo** [reply]
Good point.  The 'what' is to identify dupes from other clients.  The 'how' is negotiable.  ;-)

**Reply by mbridak** [reply]
I've made the changes to the fdserver and fdlogger and pushed the changes to PyPi.

**Reply by kybrjo** [reply]
It works beautifully with clients running on Windows, Mac, Linux Mint & Raspberry Pi (with the server running on the pi).

Thank you!!!
